wcag-tool/chrome-extension/src/components/editor/OverlayButton.jsx

```diff
@@ -45,11 +38,8 @@ const OverlayButton = function () {
       <FloatButton
         tooltipText="Exit Selection Mode"
         toolTipPosition="bottom-start"
-        onClick={() => {
-          dispatch(setListSliderIsOpen(true))
-          dispatch(setSelectElement(false))
-        }}
-        hidden={!selectElement}
+        onClick={openListSlider}
+        hidden={!elementSelecorIsOpen}
```

Comment:
```suggestion
        hidden={elementSelecorIsOpen}
```
ik gok dat dit de oplossing is. Deze knop moet weergeven worden wanneer hij wel open is.